src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/WritableLaunchSettings.cs

```diff
@@ -42,19 +42,20 @@ public WritableLaunchSettings(ILaunchSettings settings)
                     else
                     {
                         string jsonString = JsonConvert.SerializeObject(value, Formatting.Indented, jsonSerializerSettings);
-                        object clonedObject = JsonConvert.DeserializeObject(jsonString, value.GetType());
+                        // TODO: Bug in nullable flow analysis her?
```

Comment:
I think this is also [https://github.com/dotnet/roslyn/issues/31803](https://github.com/dotnet/roslyn/issues/31803)